superset/translations/ru/LC_MESSAGES/messages.json

```diff
@@ -5,5682 +5,6034 @@
       ],
-      "Superset encountered an unexpected error.": [
-        "Суперсет столкнулся с неожиданной ошибкой."
+      "%(message)s\nThis may be triggered by: \n%(issues)s": [
+        "%(message)s\nВозможные причины: \n%(issues)s"
       ],
-      "The username provided when connecting to a database is not valid.": [
-        "Имя пользователя, указанное при подключении к базе данных, недействительно"
+      "%(name)s.csv": ["%(name)s.csv"],
+      "%(name)s.pdf": ["%(name)s.pdf"],
+      "%(object)s does not exist in this database.": [
+        "%(object)s не существует в этой базе данных"
       ],
-      "The password provided when connecting to a database is not valid.": [
-        "Неверный пароль для базы данных."
+      "%(other)s charts will appear here": [
+        "%(other)s диаграмм появятся здесь"
       ],
-      "Either the username or the password is wrong.": [
-        "Неверное имя пользователя или пароль"
+      "%(other)s dashboards will appear here": [
+        "%(other)s дашбордов появятся здесь"
       ],
-      "Either the database is spelled incorrectly or does not exist.": [
-        "Неверное или несуществующее имя базы данных."
+      "%(other)s recents will appear here": [
+        "%(other)s недавних появятся здесь"
       ],
-      "The schema was deleted or renamed in the database.": [
-        "Схема была удалена или переименована в базе данных."
+      "%(other)s saved queries will appear here": [
+        "%(other)s сохраненных запросов появятся здесь"
       ],
-      "User doesn't have the proper permissions.": [
-        "У пользователя нет надлежащего доступа."
+      "%(prefix)s %(title)s": ["%(prefix)s %(title)s"],
+      "%(report_type)s schedule frequency exceeding limit. Please configure a schedule with a minimum interval of %(minimum_interval)d minutes per execution.": [
+        ""
       ],
-      "One or more parameters needed to configure a database are missing.": [
-        "Один или несколько параметров, необходимых для настройки базы данных, отсутствуют"
+      "%(rows)d rows returned": ["Получено строк: %(rows)d"],
+      "%(subtitle)s\nThis may be triggered by:\n %(issue)s": [
+        "%(subtitle)s\nВозможные причины:\n %(issue)s"
       ],
-      "The submitted payload has the incorrect format.": [
-        "Загруженные данные имеют некорректный формат."
+      "%(type)s File": ["Файл %(type)s"],
+      "%(validator)s was unable to check your query.\nPlease recheck your query.\nException: %(ex)s": [
+        "%(validator)s не смог проверить ваш запрос.\nПожалуйста, перепроверьте его вручную.\nОшибка: %(ex)s"
       ],
-      "The submitted payload has the incorrect schema.": [
-        "Загруженные данные имеют некорректную схему."
+      "%s Error": ["%s Ошибка"],
+      "%s PASSWORD": [""],
+      "%s SSH TUNNEL PASSWORD": [""],
+      "%s SSH TUNNEL PRIVATE KEY": [""],
+      "%s SSH TUNNEL PRIVATE KEY PASSWORD": [""],
```

Comment:
it shouldn't be empty I believe

Reply:
@artemonsh, thank you for the feedback!

Right, there are still quite a bunch of empty strings. Translating, I was leaving particular string as is if:
* I haven't encountered it in the UI and wasn't sure about the context.
* The string could be a common term that shouldn't be translated.
* It looked like some low-level stuff that, when translated, could confuse end user. For instance, I have intentionally left lines 81-84 that you have mentioned. I haven't stumbled upon them in the UI and, given that they are uppercase, I thought that I could break something if I provide a translation. That was the case when someone among previous contributors has translated "DELETE" string (or some other one like that) and was asked to leave it as is.
* I wasn't sure how to properly translate a string because I am not familiar with related functionality and didn't want to mess up.

I totally agree there are a lot more work to do. At this point my goal was to provide at least something I was confident with to improve user experience, not the full localization. As the next step, I was planning to check specific chart settings, fill the gaps there and straighten up the terminology used. But I think it's better to deliver that in future PRs.

Regarding what word to use as "chart" term. I totally agree that's one of the core terms and it should be picked very carefully. For reference, Ozhegov's dictionary gives this definition to the term:

> ГРАФИК - диаграмма, изображающая при помощи кривых количественные показатели движения, состояния чего-н.
> ГРАФИК is a diagram that depicts quantitative indicators of movement, state of something **with curves**.

Diagram is described as follows:

> ДИАГРАММА - Графическое изображение соотношения каких-н. величин.
> ДИАГРАММА is a graphical representation of the relationship of some quantities.

Come on, "Диаграмма" is just 3 characters longer than "График", but that's much more suitable :)

Yeah, Power BI is very popular, it's my daily workhorse as well. Just checked the naming in v2.126.927.0 (February 2024), and they used "Визуализации" ("Visualizations") for visual elements of dashboards, which is even longer.

<img width="235" alt="PBI" src="https://github.com/apache/superset/assets/7242338/95ab8e32-648d-44c0-a703-018eda7172ab">

Of course, trying to incorporate new naming into Superset, I have checked that the word fits the UI space. Looked quite nice and I haven't seen any elements that were broken on my 13'' laptop screen. Let me show you some examples.

**"Charts" screen**
<img width="1440" alt="Charts" src="https://github.com/apache/superset/assets/7242338/f92e9593-28fd-4496-a43b-96e12e43be61">

**"Create chart" dialog**
<img width="1440" alt="Create chart" src="https://github.com/apache/superset/assets/7242338/3ad6fae4-6f5a-4822-946d-e1fd82c32706">

**Chart editor**
<img width="334" alt="Chart editor" src="https://github.com/apache/superset/assets/7242338/8c3cd4ff-a9cb-4506-93ab-d7ea4e7fad2b">

**Picking a chart in Dashboard editor**
<img width="373" alt="Dashboard editor" src="https://github.com/apache/superset/assets/7242338/c100c108-ce0a-4ef2-9639-8b7a64d37fd8">

If there are some other spots worth checking, please tell me.

Reply:
Where can I see these in the UI to get some context?

Reply:
I've found these strings in `superset-frontend/src/components/ImportModal/index.tsx`. Seems they are related to "Import database from file" dialog. But I can't get this thing to work for my local dev installation. I've exported one of my databases and tried to import it, but I'm receiving this error:

```
An error occurred while importing database: Error importing database. Please re-export your file and try importing again
```

Should I translate these strings? If so, may I have a screenshot of related UI to do that properly? @rusackas @artemonsh

Reply:
Without any further info, I've translated these strings, but marked them as a draft.